client/src/components/Homepage/index.js

```diff
@@ -18,16 +18,14 @@ import ModalBox from '../Modalbox/ModalBox'
 import '../../fonts/JetBrainsMono[wght].ttf'
 import './Homepage.css'
 
+//constants
+const {SOCKET,MODES,DEFAULT_CODE,DOCS,GITHUB_URL} = require('../../constants.js')
```

Comment:
Do not destructure the CONSTANTS while import. Suppose in future, we need to add more constants then the variable has to be imported separately as its restructured.
So can you please change all the constant imports to `const CONSTANTS=require('../../constants.js')`